examples/nvme/perf/perf.c

```diff
@@ -1108,7 +1155,10 @@ int main(int argc, char **argv)
 
 	print_stats();
 
+FAIL:
+	dissociate_worker_from_ns();
```

Comment:
I would suggest just adding two new functions:

unregister_namespaces():  walks the g_namespaces list and frees all of the entries
unregister_workers():  walks the g_workers list - for each worker, free all of its ns entries, then free the worker itself

Thanks for taking time to provide this clean-up patch for the perf tool!